service/userService_test.go

```diff
@@ -0,0 +1,104 @@
+package service
+
+import (
+	"context"
+	"errors"
+	"github.com/murilogilfelpeto/ps-tag-onboarding-go/mocks"
+	"github.com/murilogilfelpeto/ps-tag-onboarding-go/service/models"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestSaveUser(t *testing.T) {
+	t.Run("Persist User", func(t *testing.T) {
+		mockRepo := &mocks.Repository{}
+
+		user, _ := models.NewUser("f7d2ea4b-a4d0-4103-9c63-55ec7977e4d1", "John", "Doe", "john.doe@email.com", 36)
```

Comment:
handling errors appropriately, especially in testing scenarios, is a good practice. Ignoring errors in tests can lead to undetected issues.

Reply:
For this scenario there is no expected error and if there is one the user will return empty, failing the test. Do you think that this should be validated?